src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -67,12 +67,15 @@ public JsonAdaptedClient(@JsonProperty("name") String name, @JsonProperty("phone
     public JsonAdaptedClient(Client source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
-        email = source.getEmail().value;
-        address = source.getAddress().value;
-        Optional<Birthday> sourceBirthday = source.getBirthday();
-        birthday = sourceBirthday.isEmpty()
+        email = source.getEmail().isEmpty()
             ? ""
-            : sourceBirthday.get().toString();
+            : source.getEmail().get().toString();
```

Comment:
a slight difference in that after `get` on the Optionals, toString is used vs in essential fields like name, direct accessors are used. 

this is not an issue from ur end, but seems to me like the original classes `Name` etc shouldn't be making their attributes public.